packages/RadioGroup/src/RadioGroup.tsx

```diff
@@ -20,7 +22,34 @@ const baseStyle = css`
   padding: 5px;
 `;
 
-export default class RadioGroup extends PureComponent {
+function isRadioElement(
+  element: React.ReactNode,
+): element is React.ReactElement<RadioProps, typeof Radio> {
+  return (
```

Comment:
Boy I want optional chaining in JS... Is there any reason we can't do:

```js
return element && element.type && element.type.displayName === 'Radio'
```

Should be easier to read and accomplish the same thing.

One day we can make it something like...
```js
return element?.type?.displayName === 'Radio'
```

I can dream on that last part 😁

Reply:
TS wouldn't like that. If `element` is `null` you'd be returning `null` when it wants a boolean, and if it's a string or some other non-object thing that's a valid React node, then `.type` doesn't exist on it.

Reply:
Right, so it would have to be something like:

```js
return !!(element && element.type && element.type.displayName === 'Radio')
```

Good point, I defer to you then 🤷‍♂

Reply:
And that would only solve the "null" issue, you'd still have to convince it that there really is a `type` key on there :)

Reply:
I tested it out, it also solves for returning undefined (when it's a string, or Array for instance).

Reply:
Yeah, TS seems to be generally of the opinion that having all unknown properties on an object come back as `undefined` was a mistake.